app.controllers/article.controllers.js

```diff
@@ -0,0 +1,12 @@
+const { fetchArticleById } = require("../app.models/articles.models");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticleById(article_id)
+    .then((article) => {
+      res.status(200).send(article);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
- `getArticleById` is a well named controller function
- Good work extracting the `article_id` off the params and storing it into a variable
- For the response, you should be sending a key./value pair back, currently you're just sending back the object on no key, this is not convention. It should be something like `.send({article: article})`. Please get into the habit of this convention.
- Good work implementing the catch block and next function to allow the error handling middleware do it's job